Gemfile.lock

```diff
@@ -24,22 +24,22 @@ GIT
 
 GIT
   remote: https://github.com/CocoaPods/Nanaimo.git
-  revision: aa4d0c50772c4392039e70a88bd9cc944064f0ad
+  revision: cdd75d07a93dd51680ad75c78a40d813e67bd968
```

Comment:
@Whirlwind Could you undo the changes to this file?

Reply:
I upgrade the `xcodeproj` gem and it have a newer dependency for `nanaimo`( from 0.2.5 to 0.2.6), so the `nanaimo` is updated.

Reply:
Ah ok, makes sense